tensorflow/contrib/layers/python/layers/layers_test.py

```diff
@@ -3688,7 +3688,17 @@ def test_1d_invalid_input(self):
                                    'rank of x must be at least 2 not: 1'):
         x = constant_op.constant([[]], shape=[0])
         _layers.legacy_fully_connected(x, 2, activation_fn=nn_ops.softmax)
-
-
+  
+class zero_debias_moving_mean(test.TestCase):
+  
+  def Error_in_tf.contrib.layers.batch_norm_when(self):
+    import tensorflow as tf
```

Comment:
In internal tests, you have to import the modules individually, you cannot import tensorflow here.  So you should use something like:

a = array_ops.placeholder(dtypes.float32, ...)

Take a look at the existing batch norm tests and use that as a guide for how to write the test where center=False